contracts/infrastructure/salary/PayrollClerk.sol

```diff
@@ -56,32 +56,35 @@ contract PayrollClerk is PayrollClerkStorage {
     return workers.length;
   }
 
+  function getPrice(address token) public view returns (uint256) {
```

Comment:
use `internal` instead of `public`
public function call cost much more than internal

Reply:
I have updated it.